pom.xml

```diff
@@ -2070,6 +2070,7 @@
                         </python>
                         <markdown>
                             <includes>
+                                <include>**/README.md</include>
```

Comment:
It is over-matched and may cause issues.
https://github.com/apache/kyuubi/actions/runs/4169081487/jobs/7216619947#step:4:121483

Reply:
Let me change the pattern into parts, `dev/**/README.md`,`docker/playground/README.md`, `extensions/**/README.md`, `*/README.md`